### PR TITLE
Add cw to chain schema

### LINKED ISSUE #419: Add cosmwasm to chain.json?
Add cosmwasm to chain.json? This can help establish a set of features for Keplr::Suggest Chainlist.

Could do something like:
`"cosmwasm_version": 0.24`
or
`"cosmwasm_enabled": true` 

How to we find the version of comswasm a chain is using? or How do we find what the latest version of cosmwasm is?

Do we need to allow listing a specific set of features?

Should this go under the codebase object in chain.json?

### PR DESCRIPTION
**Pull request by JeremyParish69**
adds cosmwasm version and enabled status to chain.json files
resolves #419 

within codebase. to go below binaries, and below tendermint and cosmos sdk version once those are added.

-"cosmwasm_version": string
e.g., "0.24" (does not have v- prefix, like v0.24--should it?)
only major and minor version

-"cosmwasm_enabled": boolean
e.g., true

### CODE REVIEW

File: chain.schema.json [hunk 1]
```diff
@@ -106,6 +106,12 @@
                             "format": "uri"
                         }
                     }
+                },
+                "cosmwasm_version": {
+                    "type": "string"
+                },
+                "cosmwasm_enabled": {
+                    "type": "boolean"
```
[the remark this answers is not in the document]
Comment (JeremyParish69) [reply chain.schema.json:114]: yes

**Review by ValarDragon — APPROVED** [review]

File: chain.schema.json (hunk 1 above)
Comment (ValarDragon) [inline chain.schema.json:114]: I think this will later need to be an enum to encapsulate permissioned

**Comment by mccallofthewild** [comment]
[Reposting](https://github.com/cosmos/chain-registry/issues/419#issuecomment-1176887671) here

This is a good idea but vm-specific metadata shouldn't be top level in the chain schema. Operate on the assumption that all chains will implement IBC.

Per @ValarDragon 
> I think this will later need to be an enum to encapsulate permissioned

Let's just implement it correctly to start with. The purpose of this repository is to design a schema for the CNS.
 
 I would support something like this:
```json
{
  [...]
  "contract_environments": [
    {
      "permissionless": false,
      "virtual_machine": "evm",
      "version": "v0.13.2"
    }
  ]
  [...]
}
```
where `virtual_machine` is an enum/union of
```
"cosmwasm" | "evm" | "zoe" | "gno" | ...
```

**Comment by JeremyParish69** [comment]
@mccallofthewild , I had it under codebase, not at top level, but I like how more generalized your suggestion is. I wouldn't have the capacity to produce an exhaustive enum, should we start with the ones that we know we'll need right away (I'm guessing that's cosmwasm and evm, but I could be missing some)?

Also, I notice that this omits whether cosmwasm is enabled. Might like to then propose an "enabled" property--or, if it needs to be more generic, a "status" property--at least some way we can show whether cosmwasm is enabled.

**Review by ankurdotb — APPROVED** [review]

**Pull request merged**